plugins/janus_audiobridge.c

```diff
@@ -1321,6 +1327,10 @@ int janus_audiobridge_init(janus_callbacks *callback, const char *config_path) {
 		if(!notify_events && callback->events_is_enabled()) {
 			JANUS_LOG(LOG_WARN, "Notification of events to handlers disabled for %s\n", JANUS_AUDIOBRIDGE_NAME);
 		}
+		janus_config_item *fec = janus_config_get_item_drilldown(config, "general", "use_fec");
+		use_fec = fec && fec->value && janus_is_true(fec->value);
+		JANUS_LOG(LOG_VERB, "FEC: %s\n", use_fec ? "enable" : "disable");
```

Comment:
Why doing a global setting for FEC? Wouldn't it be better to have a per room, or even better a per-participant, FEC property? Ideally, this could be done depending on the SDP negotiation: if the user offers FEC in the SDP, then we enable it for them, otherwise we don't. What do you think about that?

As a side note, I guess the log line there is temporary and just for debug purposes?

Reply:
I'm agree that the best solution is to only use SDP negotiation information and according to the value associate to FEC usage perform operation. 
I will do it in a second step if you want.